azure-pipelines.yml

```diff
@@ -137,28 +137,11 @@ steps:
         **/*Tests*/Log/*.log
     TargetFolder: '$(Build.ArtifactStagingDirectory)'
   condition: succeededOrFailed()
-
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish copied artifacts'
-  inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
-    ArtifactName: 'drop'
-    publishLocation: 'Container'
-  condition: succeededOrFailed()
```

Comment:
oh no, these artifacts(log and screens) should be published unconditionally, not just for master source branch

Reply:
Why do you need them?

Reply:
logs can help to detect failure reasons, especially when previous test did smth which caused the failure of the next test

Reply:
changes reverted